docs2/site/docs/migrations/migration5.md

```diff
@@ -622,3 +693,25 @@ invisible to the caller and lead to hardly detected bugs, the method name has be
 Previously the settings class used was `Newtonsoft.Json.JsonSerializerSettings`. Now the class
 is `GraphQL.NewtonsoftJson.JsonSerializerSettings`. The class inherits from the former class,
 but sets the default date parsing behavior set to 'none'.
+
+### 29. `SubscriptionDocumentExecuter` and `.AddSubscriptionDocumentExecuter()` have been deprecated.
+
+While these can continue to be used for the lifetime of v5, it is now suggested to use the
+`.AddSubscriptionExecutionStrategy()` builder method instead:
+
+```csharp
+// v4
+services.AddGraphQL()
+   .AddSchema<StarWarsSchema>()
+   .AddSubscriptionDocumentExecuter();
+
+// v5
+services.AddGraphQL(builder => builder
+   .AddSchema<StarWarsSchema>()
+   .AddSubscriptionExecutionStrategy());
+```
+
+For more details on the new approach to execution strategy selection, please review the new feature above titled:
+
+> The `ExecutionStrategy` selected for an operation can be configured through `IGraphQLBuilder`
```

Comment:
Markdown supports local links. We can use it for such things + also to provide some sort of table of contents.

Reply:
migration2 : 76 lines
migration3: 479 lines
migration4: 704 lines
migration5: 716 lines and continues to grow 😮

Reply:
I know but since the migration file isn’t finished the link would likely break. This is the kind of thing I would do during cleanup immediately before release. 

The documentation for the new AutoRegistering graph types will likely all be moved into the main docs with a link pointing to the new functionality. This will probably save a lot of lines of text. Again, we should do documentation cleanup once the functionality has settled. 

I do prefer to keep documenting as the changes are being made. For the longest time the only thing holding back a 3.x release was the fact that no docs, of any sort, were added while the changes were being made. For instance I know little about directives. So if I were to issue a release without you having made docs for the changes made to directives, that would be a problem. But if we all make notes during the changes it is easy to reorder or reformat the migration docs at the end.